### PR TITLE
remove allure attaches for passed tests by title

### PR DESCRIPTION
**Pull request by eoff**

### CODE REVIEW

File: allure-java-adaptor-api/src/test/java/ru/yandex/qatools/allure/events/RemoveAttachmentsByTitleEventTest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    @Test
+    public void removeAttachmentsByTitleRegexTest() throws Exception {
+        List<String> files = new ArrayList<>(asList(resultsDirectory.list()));
+        files.remove(toRemove.getSource());
+
+        new RemoveAttachmentsByTitleEvent("first.*").process(root);
+
+        assertThat(resultsDirectory, notContains(toRemove.getSource()));
+        for (String file : files) {
```
Comment (lanwen) [inline allure-java-adaptor-api/src/test/java/ru/yandex/qatools/allure/events/RemoveAttachmentsByTitleEventTest.java]: matcher `everyItem` exists for that

**Comment by lanwen** [comment]
think it should be some kind of allure-extension

**Comment by eoff** [comment]
this was a hotfix at some moment and this is just a simple fix to enable attachment deletion by title. don't want to change a lot of code structure as baev@ is currently refactoring it. actually he's seen this pr already

**Comment by baev** [comment]
@eoff thanks for the contribution. 

Could you please fix comments and squash your commits into one. See http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request and http://stackoverflow.com/questions/14534397/how-to-squash-all-my-commits-into-one-github

**Comment by baev** [comment]
closed due to no activity

**Pull request closed without merging**